cms/settings/base.py

```diff
@@ -544,32 +541,43 @@
 
 
 # Email settings
-# We use SMTP to send emails. We typically use transactional email services
-# that let us use SMTP.
-# https://docs.djangoproject.com/en/2.1/topics/email/
+# https://docs.djangoproject.com/en/stable/topics/email/
+
+# Use fixed strings in case import paths move
+match env.get("EMAIL_BACKEND_NAME", "").upper():
+    case "SES":
+        EMAIL_BACKEND = "django_ses.SESBackend"
```

Comment:
```suggestion
        # See https://github.com/django-ses/django-ses
        EMAIL_BACKEND = "django_ses.SESBackend"
```
I think Dan suggested this in https://github.com/ONSdigital/dis-wagtail/pull/224#discussion_r2123930331 but it got lost. I think it's worth keeping it there to avoid make it clear what we're using and allow quick reference to the env vars required by this library.